android.md

```diff
@@ -34,6 +34,9 @@ class Person {
   
   // region Instance Methods
   // endregion Instance Methods
+  
+  // region interface 
```

Comment:
@nimesh7424 We don't need a section for interface. Mostly interfaces are public and are declared in their separate files. Let's check our project and see if there is a valid use-case for this.